src/shared/components/login/socialLogin.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import axios from 'axios';
+import config from 'config/environment';
+import Section from 'shared/components/section/section';
+import Form from 'shared/components/form/form';
+import FormZipCode from 'shared/components/form/formZipCode/formZipCode';
+import FormPassword from 'shared/components/form/formPassword/formPassword';
+import FormButton from 'shared/components/form/formButton/formButton';
+import styles from 'scenes/home/informationForm/informationForm.css';
+import _ from 'lodash';
+import * as CookieHelpers from '../../utils/cookieHelper';
+
+let firstName;
```

Comment:
I see you've got unbound functions and these variables outside of the component scope. This is what we call a code smell, because it's not idiomatic of the framework. I'm still trying to grok what's going on with this file. Can you describe the purpose of socialLogin.js to me? I can better guide some changes with your input.